### DISCUSSION TITLE
How to avoid having to display the media list

### DISCUSSION THREAD

**Discussion by Fakhri17** [awcodes/filament-curator#554] [Q&A]
![image](https://github.com/user-attachments/assets/53e4ffb7-b521-4988-8d02-57c5a9fb8821)

Is it possible to not need to display the media list when uploading media?

**Comment by awcodes** [comment]
Not really possible, the whole point of this is to be able select previously uploaded media.

**Reply by Fakhri17** [reply]
How do I get only media that has been uploaded by that user?

**Reply by awcodes** [reply]
You define a PathGenenerator for the user. There's one included with the plugin. Then limit the directory to the user id.
